index.js

```diff
@@ -75,8 +105,15 @@
 //     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
 //   }
 // ]
+// function printNameAndPhones(user) {
+//   for (let i = 0; i < user.length; i++) {
+//     console.log(user[i].name)
+//     console.log(user[i].phones.cell)
+//     console.log(user[i].phones.office)
+//   }
+// }
```

Comment:
Nice job using dot notation for objects. Alternatively, you could have used a .forEach instead of a for loop.